tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -0,0 +1,156 @@
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("3", "3", "2", "1");
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm the breakdown question based on answer value source with valid answers before I can continue", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("2", "2", "1", "1");
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("0", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(BreakdownAnswerPage.pageName);
+
+      $(BreakdownAnswerPage.submit()).click();
+      expect($(BreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 15");
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm the breakdown question based on answer value source with valid answers before I can continue", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("2", "2", "1", "1");
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("6", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("1", "1", "1", "1");
+
+      expect(browser.getUrl()).to.contain(SecondBreakdownAnswerPage.pageName);
+
+      $(SecondBreakdownAnswerPage.submit()).click();
+      expect($(BreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 9");
+    });
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 5 into the total", () => {
+    it("When I continue and enter 3 in each breakdown field, Then I should see a validation error", () => {
+      $(TotalAnswerPage.total()).setValue("5");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      expect($(BreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 5");
+    });
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 5 into the total", () => {
```

Comment:
Are these two not already covered?

Reply:
do you mean in this spec? We do have validation passing when the value total is correct but not when it's less or more than total value entered before. Second one is for value source validation error, first for answer source.

Reply:
No I mean these are technically already covered by the other, but happy to keep.